sound/soc/sof/pm.c

```diff
@@ -146,6 +146,23 @@ static int sof_restore_pipelines(struct snd_sof_dev *sdev)
 	list_for_each_entry_reverse(dai, &sdev->dai_list, list) {
 		struct sof_ipc_reply reply;
 		struct sof_ipc_dai_config *config = dai->dai_config;
+		struct snd_sof_dai *pdai;
+		int restored = false;
+
+		list_for_each_entry_reverse(pdai, &sdev->dai_list, list) {
```

Comment:
@dragosht the patch looks good. Could you please add a comment here to make it clear that we're preventing sending the config for the same BE DAI again?

Reply:
@ranj063 Sure. Thanks.